vagrant/boxes.d/99-local.yaml.example

```diff
@@ -69,24 +87,24 @@ centos7-foreman-bats-ci:
     playbook: 'playbooks/bats_pipeline_foreman_nightly.yml'
     group: 'bats'
 
-centos8-dynflow-devel:
-  box: centos8-stream
+almalinux8-dynflow-devel:
+  box: almalinux8
   memory: 512
```

Comment:
I really doubt an EL8 box will function with 512 MB RAM, but probably outside of this PR's scope